<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>469</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>469</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Muladhara Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo2.png</normaloff>logo2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 235, 235);</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>271</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>271</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/logo2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_login">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>239</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	background-color: rgb(220, 57, 62);
	
	color: rgb(77, 11, 17);
	border-radius: 15px

}

QPushButton:hover{

	
	background-color: rgb(77, 11, 17);
	
	color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_senha">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Senha</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usuario">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Usuario</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	background-color: rgb(0, 0, 0,0.1);
	border-radius: 20px



}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_usuario</zorder>
  <zorder>lineEdit_senha</zorder>
  <zorder>pb_login</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
